.coderabbit.yml

```diff
@@ -0,0 +1,30 @@
+language: "en"
+early_access: false
+reviews:
+    request_changes_workflow: false
+    high_level_summary: true
+    poem: false
```

Comment:
:thinking: now I'm curious as to what this is :D

Reply:
and you set it to false! :scream:

Reply:
I know, I know... It was tough decision... :)